src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIResponsesChatClient.cs

```diff
@@ -259,10 +301,48 @@ ChatResponseUpdate CreateUpdate(AIContent? content = null) =>
                     responseId = createdUpdate.Response.Id;
                     conversationId = options?.StoredOutputEnabled is false ? null : responseId;
                     modelId = createdUpdate.Response.Model;
+                    latestResponseStatus = createdUpdate.Response.Status;
+                    goto default;
+
+                case StreamingResponseQueuedUpdate queuedUpdate:
+                    createdAt = queuedUpdate.Response.CreatedAt;
+                    responseId = queuedUpdate.Response.Id;
+                    conversationId = options?.StoredOutputEnabled is false ? null : responseId;
+                    modelId = queuedUpdate.Response.Model;
```

Comment:
Are these necessary? I was under the impression they'd all come as part of the CreatedUpdate; same question for the other cases bellow (in progress, incomplete, etc.)